<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSADistanceColumnSettingsForm</class>
 <widget class="QWidget" name="MSADistanceColumnSettingsForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QWidget" name="distnanceColumnSettingsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="showDistancesColumnCheck">
        <property name="text">
         <string>Show distances column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="optionsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="algoLabel">
           <property name="text">
            <string>Distance algorithm:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="algoComboBox"/>
         </item>
         <item>
          <widget class="QCheckBox" name="autoUpdateCheck">
           <property name="text">
            <string>Automatic updating</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="profileGrope">
           <property name="title">
            <string>Profile mode</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="margin">
             <number>5</number>
            </property>
            <item>
             <widget class="QRadioButton" name="countsButton">
              <property name="text">
               <string>Counts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="percentsButton">
              <property name="text">
               <string>Percents</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="excludeGapsCheckBox">
              <property name="text">
               <string>Exclude gaps</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="refSeqWarning">
           <property name="text">
            <string>&lt;empty&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
